### PR TITLE
H8

### PR DESCRIPTION
**Pull request by gabrieldiaz89**
Viejo

### CODE REVIEW

File: app/Http/Controllers/AuthorController.php [hunk 1]
```diff
@@ -25,5 +25,26 @@ public function store(Request $request)
...
+
+        Author::destroy($id);
+        /*
+        !!No se encuentran los metodos!!
+        $author= Author::find($id);
+        $author->delete();
+        */
+
+
+        return "ok";
```
Comment (eb-octavio) [inline app/Http/Controllers/AuthorController.php]: Can we return something like https://github.com/avgvsto/moreno/blob/d1216256d86e1c999fdd320961e3c326a9407433/app/Http/Controllers/AuthorController.php#L37
?

File: app/Http/Controllers/AuthorController.php [hunk 2]
```diff
@@ -25,5 +25,26 @@ public function store(Request $request)
 
         return response()->json(['id' => $author->id]);
     }
+    public function destroy($id)
+    {
+
+        Author::destroy($id);
```
Comment (eb-octavio) [inline app/Http/Controllers/AuthorController.php]: Before destroy the author we should check the id exists in the database.

File: app/Http/Controllers/AuthorController.php [hunk 3]
```diff
@@ -25,5 +25,26 @@ public function store(Request $request)
...
+        $author->delete();
+        */
+
+
+        return "ok";
+    }
+
+    public function index()
+    {
+        $author=Author::all();
```
Comment (eb-octavio) [inline app/Http/Controllers/AuthorController.php]: Please call this variable `$authorS`, use `json` too:
`return response()->json($authors);`

File: app/Http/Controllers/AuthorController.php (hunk 1 above)
Comment (avgvsto) [inline app/Http/Controllers/AuthorController.php]: Can we return something like
https://github.com/avgvsto/moreno/blob/d1216256d86e1c999fdd320961e3c326a9407433/app/Http/Controllers/AuthorController.php#L37
??

File: app/Http/Controllers/AuthorController.php [hunk 4]
```diff
@@ -25,5 +25,26 @@ public function store(Request $request)
 
         return response()->json(['id' => $author->id]);
     }
+    public function destroy($id)
+    {
+
```
Comment (avgvsto) [inline app/Http/Controllers/AuthorController.php]: Before delete the user, we should check that the user exists in the database.

File: app/Http/Controllers/AuthorController.php [hunk 5]
```diff
@@ -25,5 +25,26 @@ public function store(Request $request)
...
+        return "ok";
+    }
+
+    public function index()
+    {
+        $author=Author::all();
+
+
+        return $author;
```
Comment (avgvsto) [inline app/Http/Controllers/AuthorController.php]: We should return something like `response()->json($authors);`.

And the name should be $authors.

**Review by avgvsto — COMMENTED** [review]
Some comments.

**Pull request merged**